docs/general-reference/information-schema/tables.md

```diff
@@ -33,7 +33,7 @@ Each row has the following columns with information about each table.
 | size                        | DOUBLE      | The compressed size of the table. |
 | size_uncompressed           | DOUBLE      | The uncompressed size of the table. |
 | compression_ratio           | DOUBLE      | The compression ratio (`<size_uncompressed>`/`<size>`). |
-| number_of_segments          | INT         | The number of segments comprising the table. |
+| number_of_tablets           | INT         | The number of segments comprising the table. |
```

Comment:
Updated definition to match the one from SHOW TABLES, in release draft here: https://laughing-adventure-r4eyrz4.pages.github.io/general-reference/information-schema/tables.html#information-schema-for-tables

Will merge this once those notes are published.

Reply:
fixed